.github/workflows/build-commit-run-tpch.yaml

```diff
@@ -0,0 +1,41 @@
+name: Build commit and run tpch benchmarks
+
+on:
+  workflow_dispatch:
+    inputs:
+      wheel:
+        description: The wheel artifact to use
+        required: false
+        default: getdaft-0.3.0.dev0-cp38-abi3-manylinux_2_31_x86_64.whl
+      skip_questions:
+        description: The TPC-H questions to skip
+        required: false
+        default: ""
+      scale_factor:
+        description: Which scale factor to use
```

Comment:
Can you put the available configurations in the description or somewhere where we can easily see?

Reply:
Ya, I'll actually make this an option that's selectable only. Good catch.

Reply:
Updated.